core/dbschema/createdbs.sql

```diff
@@ -1,4 +1,5 @@
 CREATE DATABASE IF NOT EXISTS eb;
+CREATE DATABASE IF NOT EXISTS eb_test;
```

Comment:
In webauthn I opted not to use a separate _test db, because each environment is only used for one purpose.
In the pipeline, the eb db is only used for tests.

Is it currently so that on our dev checkout, the tests are executed in the `eb_test` and `eb` is used for development? Because than I understand.

Or was this easier for other reasons?

Reply:
It's done to be able to provision test data in your development box for tests while maintaining test data in your regular dev env.

Reply:
:+1:

Reply:
It was always possible to run tests in a separate db, but it seems that this wasn't yet implemented in Devconf.